components/vr-model.js

```diff
@@ -37,7 +36,7 @@
           update: {
             value: function() {
 
-              // Not sure what should go here...
+              // If we want to edit via dom inspector we need ot add update functions here (is my understanding).
```

Comment:
ah so update is called when the element is updated! I was wondering why I wasn't seeing it called every frame.  Makes sense now